airflow_clickhouse_plugin/sensors/clickhouse_sql_sensor.py

```diff
@@ -0,0 +1,23 @@
+from airflow.sensors.sql_sensor import SqlSensor
+from airflow_clickhouse_plugin.hooks import clickhouse_hook
+
+
+class ClickHouseSqlSensor(SqlSensor):
+    def __init__(
+        self,
+        clickhouse_conn_id: str = "clickhouse_default",
+        database=None,
+        *args,
+        **kwargs
```

Comment:
Please list arguments explicitly: `sql`, `parameters`, etc.

`*args`/`**kwargs` leads to unexpected behaviour. We would like to have control over the arguments passed to the initializer. This also makes hard to understand the interface of the sensor.